tests/linktype_name/test.yaml

```diff
@@ -0,0 +1,12 @@
+requires:
+  min-version: 8
+
+args:
+- -k none --set outputs.1.eve-log.types.0.alert.packet=yes --set vars.address-groups.EXTERNAL_NET=any
```

Comment:
Could we just change the rule to use `any` instead of `EXTERNAL_NET` ?

Reply:
WIll do.